docs/guides/security/monitoring/security-auditing-with-lynis/index.md

```diff
@@ -0,0 +1,161 @@
+---
+author:
+  name: Hackersploit
+description: 'Learn to perform a security audit on your system using Lynis.'
```

Comment:
Can probably add "an extensible security audit tool" to the end of this description and og_description just to explain what Lynis is.